src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/Object.ReadTests.cs

```diff
@@ -655,5 +656,21 @@ public static void TooLittleJsonFails(string json)
 
             Assert.Equal(0, reader.BytesConsumed);
         }
+
+        // Regression test for https://github.com/dotnet/runtime/issues/61995
```

Comment:
nit\style: I prefer not linking issues since it is unnecessary if the test name and optional comments explain what is being tested, plus GitHub history can be used to determine when\why the test was added.